simple-promise-queue/.gitignore

```diff
@@ -57,4 +57,5 @@ typings/
 # dotenv environment variables file
 .env
 
-dist
\ No newline at end of file
+dist
+yarn.lock
```

Comment:
did you intentionally remove the lock file?

Reply:
yes, because it's a lib - I suppose it is desirable to develop on the latest available deps according to package.json. And deps versions locked in yarn.lock are not carried over to the release package - when you use this library as a dependency in other project, they do not matter, sub-dependencies are resolved from the lib's package.json

Reply:
alright! thanks for the explanation!